README.md

```diff
@@ -1 +1,77 @@
-# ontotext-yasgui
\ No newline at end of file
+# ontotext-yasgui-web-component
+
+Wrapper project for maintaining "ontotext-yasgui" custom component. It contains three subprojects:
+1. Yasgui;
+2. ontotext-yasgui;
+3. cypress.
+
+## [Yasgui](https://github.com/TriplyDB/Yasgui)
+It contains the source code of yasgui. After build of project yasgui.min.js and yasgui.min.css artefacts will
+be copied into "ontotext-yasgui".
+
+## [ontotext-yasgui](ontotext-yasgui/README.md)
+This is main project of "ontotext-yasgui". In this project we will customize yasgui for Ontotext needs.
+
+## [cypress](cypress)
+It contains e2e tests.
+
+# Approach for the component implementation
```

Comment:
This should be in the developers-guide. Maybe here we should just link to it here.